.changeset/smooth-goats-agree.md

```diff
@@ -0,0 +1,49 @@
+---
+'astro': minor
+---
+
+Add support for generating multiple widths when using the Image component and a Picture component for supporting multiple formats.
+
+## `srcset` support
+
+Two new properties have been added to `Image` and `getImage`: `densities` and `widths`.
+
+These props can be used to generate a `srcset` attribute with multiple sources. For example:
+
+```astro
+---
+import { Image } from "astro";
+import myImage from "./my-image.jpg";
+---
+
+<Image src={myImage} width={myImage.width / 2} densities={[2]} alt="My cool image"  />
+```
+
+```html
+<img src="..." srcset="... 2x, ... 3x" alt="My cool image" />
+```
+
+## Picture component
+
+The `Picture` component can be used to generate a `<picture>` element with multiple sources. It can be used as follow:
```

Comment:
```suggestion
The experimental `<Picture />` component can be used to generate a `<picture>` element with multiple `<source>` elements. 

The example below uses `format` property to generate a `<source>` in each of the specified image formats:
```
I kind of like making "sources" code here, to distinguish the tag from "multiple sources" sounding like "original sources/locations/files" etc. unless there's a good reason not to do this! (And if "elements" isn't the best word, pick another word like tag or attribute or whatever is technically accurate here. It's also ok just to pluralize as `<source>`s, but that doesn't always visually look the best.)

Also, should probably be re-upping experimental, right?  :sweat_smile: